apps/frontend/src/network/users/__tests__/UserList.test.tsx

```diff
@@ -18,8 +20,22 @@ jest.mock('../../teams/api');
 jest.mock('../../groups/api');
 
 const mockGetUsers = getUsers as jest.MockedFunction<typeof getUsers>;
+const mockGetUsersWithAlgolia = getUsersWithAlgolia as jest.MockedFunction<
+  typeof getUsersWithAlgolia
+>;
+
+const renderUserList = async (opts: { algoliaEnabled: boolean }) => {
```

Comment:
I don't really like this because when the feature flag is removed we will have to change every test. Ideally they should not be changed and there should only be tests & code removed

Reply:
@kmaid That is two tests and we only need to remove the options passed in the `renderUserList` - I don't see the difference between removing setup code in the test vs removing setup options.